packages/web-app-files/src/components/Search/List.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <div class="files-search-result">
+    <no-content-message v-if="!activeFiles.length" class="files-empty" icon="folder">
+      <template #message>
+        <span v-if="!!$route.query.term" v-translate>TODO NOT FOUND MESSAGE</span>
+        <span v-else v-translate>TODO NO TERM MESSAGE</span>
+      </template>
+    </no-content-message>
+    <oc-table-files
+      v-else
+      id="files-personal-table"
```

Comment:
What about an id like `files-search-result-table`?
```suggestion
      id="files-personal-table"
```

Reply:
I'll add it to my PR, good point!

Reply:
the id isn't used, copy paste leftover. Gone